TODO.txt

```diff
@@ -1,7 +1,3 @@
-* Take a critical look at whether apptools.template should be in this project.
-  It appears to be adding a lot of dependencies the other packages do not need.
-  For now, its use is controlled through an extra.
-
```

Comment:
This file has not been edited since 2011, and it seems likely that all of these are now obsolete.  Even though it is orthogonal, should I just delete this file with this PR (I can open github issues for any of the bullets if needed, but they appear no longer relevant)?

Reply:
let's do that in a separate PR.